src/core/killall.c

```diff
@@ -154,10 +151,11 @@ static void wait_for_children(Set *pids, sigset_t *mask, usec_t timeout) {
 static int killall(int sig, Set *pids, bool send_sighup) {
         _cleanup_closedir_ DIR *dir = NULL;
         struct dirent *d;
+        int nb_kill = 0;
```

Comment:
please add a comment somewhere to the top of this function too that explains what this actually returns.

Reply:
also, maybe call this "n_killed"? (elsewhere we frequently use the `n_` prefix for variables that count stuff.